src/language-server/config/cache-busting-resolver.js

```diff
@@ -41,13 +80,19 @@ async function resolve(specifier, context, nextResolve) {
  * @returns {Promise<LoadResult>}
  */
 async function load(url, context, nextLoad) {
-  if (context.importAttributes.as !== "cachebust") {
+  const importAttributes = resolveImportAttributes(context);
+  const [as, format] = importAttributes.as.split(":");
+  if (as !== "cachebust") {
     return nextLoad(url, context);
   }
+  const contents =
+    "contents" in importAttributes
+      ? importAttributes.contents
+      : Object.keys(importAttributes)[1];
```

Comment:
It’s pretty weird here is that the second key of importAssertions is the content of the module.

Reply:
Yes, seriously. But if it works it is what it is - this only applies to a small version range and will go away at some point in the future 🤷